default schedule records differ between installation of docker-compose and harbor-helm

When installing via docker-compose, there is a default schedule record called `SYSTEM_ARTIFACT_CLEANUP` while there is no such record when installing harbor via harbor-helm.
<img width="1777" alt="Screen Shot 2022-11-29 at 16 37 34" src="https://user-images.githubusercontent.com/6709992/204481360-8066f5fe-c128-471f-8d8d-6c7dbac44588.png">
<img width="1766" alt="Screen Shot 2022-11-29 at 16 39 56" src="https://user-images.githubusercontent.com/6709992/204481372-fadb1aa7-4ad4-4728-9d10-82852f9404fe.png">

I could reproduce this issue. and get the following error in the harbor-core log
```
2022-11-30T06:25:29Z [ERROR] [/core/main.go:257]: failed to check the jobservice health status: timeout, error: retry timeout: failed to check health: Get "http://myrelease-harbor-jobservice/api/v1/stats": dial tcp 10.96.162.64:80: connect: connection refused
```
It seems that the retry might fail and the system artifact cleanup schedule will fail.

--- a/src/core/main.go
+++ b/src/core/main.go
@@ -248,7 +248,7 @@ func main() {
 		options := []retry.Option{
 			retry.InitialInterval(time.Millisecond * 500),
 			retry.MaxInterval(time.Second * 10),
-			retry.Timeout(time.Minute),
+			retry.Timeout(time.Minute * 5),
 			retry.Callback(func(err error, sleep time.Duration) {
 				log.Debugf("failed to ping %s, retry after %s : %v", url, sleep, err)
 			}),

